app/web/src/features/profile/edit/styles.ts

```diff
@@ -0,0 +1,81 @@
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
A lot of these classes are not shared. There's an argument for putting them here anyway, though (rather than being strict and only having shared classes here). I'm not really convinced either way, it hasn't come up that many times before. Maybe it's fine like this.

Reply:
I was thinking the same but I decided to not over-engineer in this specific case. There are not many styles and these two edit forms are isolated enough so it should not grow much...